### PR TITLE
Updating documentation of Education API's related to Advanced Validation feature in Microsoft School Data Sync

### PR DESCRIPTION
**Pull request by vdarbari**
Updating documentation of Education API's related to Advanced Validation feature in Microsoft School Data Sync

### CODE REVIEW

**Comment by vdarbari** [comment]
@kaubar please review

**Comment by kaubar** [comment]
@Lauragra can you please review these changes to the /beta documentation.

**Comment by Lauragra** [comment]
I reviewed the content and made some edits. Please also update the changelog.

**Comment by vdarbari** [comment]
Change log is updated.

File: concepts/changelog.md [hunk 1]
```diff
@@ -50,6 +50,16 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
 |Addition | v1.0 | Added the following new complex types: <br> [workingHours](../api-reference/v1.0/resources/workinghours.md) <br> [timeZoneBase](../api-reference/v1.0/resources/timezonebase.md) <br> [customTimeZone](../api-reference/v1.0/resources/customtimezone.md) <br> [standardTimeZoneOffset](../api-reference/v1.0/resources/standardtimezoneoffset.md) <br> [daylightTimeZoneOffset](../api-reference/v1.0/resources/daylighttimezoneoffset.md)|
 
 
+### Education APIs
+
+|Change type|Version|Description|
+|:---|:---|:---|
+|Change|Beta|Added reportableIdentifier property for [educationsynchronizationerror](../api-reference/beta/resources/educationsynchronizationerror.md).|
+|Change|Beta|Updated the response options for [uploadUrl](../api-reference/beta/api/educationsynchronizationprofile_uploadurl.md) API.|
+|Change|Beta|Updated the text for description of the [educationSynchronizationError](../api-reference/beta/resources/educationsynchronizationerror.md) resource type.|
```
Comment (Lauragra) [inline concepts/changelog.md]: Generally we use the changelog to track updates to the APIs, not updates to the text.

**Pull request merged**